### PR TITLE
Overflow improvements

### PR DESCRIPTION
**Pull request by chivchila**

### CODE REVIEW

File: src/scss/themes/_hero.scss [hunk 1]
```diff
@@ -159,6 +159,11 @@
 		text-align: center;
 	}
 
+	.o-teaser__meta
```
Comment (notlee) [inline src/scss/themes/_hero.scss]: These suggestions move the opening brace to fix the linting issue: 
```suggestion
	.o-teaser__meta {
```

File: src/scss/themes/_hero.scss [hunk 2]
```diff
@@ -159,6 +159,11 @@
...
+	 {
```
Comment (notlee) [inline src/scss/themes/_hero.scss]:
```suggestion
```

**Comment by notlee** [comment]
Thanks Olga! We looked at this together yesterday evening, here's a little context: This PR applies a word-break to all teaser headings and uses flexbox so whitespace is ignored and tags, which would fit within the width of the teaser, wrap rather than overflow or break when combined with a prefix. 

Extra markup in the x-dash implementation [o-teaser__meta-tag](https://github.com/Financial-Times/x-dash/blob/acbb34e1a12e03f1bd7192eb2b470029047e6179/components/x-teaser/src/MetaLink.jsx#L20) and [o-teaser__meta-promoted](https://github.com/Financial-Times/x-dash/blob/acbb34e1a12e03f1bd7192eb2b470029047e6179/components/x-teaser/src/Promoted.jsx#L4) which wraps `o-teaser__meta` children means this can't be released yet without breaking the x-dash implementation.

**Comment by origamiserviceuser** [comment]
o-teaser bundle size difference from 3.5.7 to c650300a2f72b3a8d55e1fa407a0c9709f92d565
css: 0.36kb increase (0.05kb/gzip)

**Comment by notlee** [comment]
This is good to release since we removed extra markup from x-dash :)

**Review by notlee — APPROVED** [review]

**Pull request merged**